src/Engine/SerializationUtils.java

```diff
@@ -0,0 +1,64 @@
+package Engine;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+
+public class SerializationUtils
```

Comment:
Why are we making changes to game load/save functionality as part of a Wally QoL PR?

Also why create this at all? It looks like it just creates an extra layer of boilerplate before passing through to the GameInstance functions anyway.

Reply:
To abstract the direct file I/O out of GameInstance, because writing an unnecessary file to test game save logic drives me batty.